spec/test_collector/minitest_plugin/trace_spec.rb

```diff
@@ -1,12 +1,17 @@
 # frozen_string_literal: true
 
 require "buildkite/test_collector/minitest_plugin/trace"
+require "minitest"
 
 RSpec.describe Buildkite::TestCollector::MinitestPlugin::Trace do
-  subject(:trace) { Buildkite::TestCollector::MinitestPlugin::Trace.new(result, history: history) }
-  let(:result) { double("Result", name: "test_it_passes", test_it_passes: nil, result_code: 'F', failure: failure, failures: [failure]) }
-  let(:failure) { double("Failure", message: message, backtrace: backtrace)}
-  let(:message) { "test for invalid character'\xC8'\n    Expected: true\n    Actual: false" }
+  subject(:trace) { Buildkite::TestCollector::MinitestPlugin::Trace.new(example, history: history) }
+  let(:example) { double("Minitest::Test", name: "test_it_passes", test_it_passes: nil, result_code: 'F', failure: failure, failures: [failure, another_failure]) }
+
+  # failure is either Minitest::Assertion or Minitest::UnexpectedError object. 
+  # ref: https://github.com/minitest/minitest/blob/master/lib/minitest/test.rb#L198
```

Comment:
If you push 'y' while on this page it'll generate a link to this line of code that is pegged to the commit sha so it won't change if the file changes, i.e.

https://github.com/minitest/minitest/blob/0984e29995a5c0f4dcf3c185442bcb4f493ed5e3/lib/minitest/test.rb#L198